src/partials/modal-rating.html

```diff
@@ -26,7 +26,24 @@ <h3 class="rating-title">Rating</h3>
                     <svg class="modal-star-icon" width="24" height="24" aria-label="modal star icon">
                         <use href="../img/sprite/sprite.svg#icon-exercises-star"></use>
                     </svg>
-                    <div class="rating-activ"></div>
+                    <div class="rating-active">
+                        <svg class="modal-star-icony" width="24" height="24" aria-label="modal star icon">
+                            <use href="../img/sprite/sprite.svg#icon-exercises-star"></use>
```

Comment:
Path should be with 1 dot. example: `<use href="./img/sprite/sprite.svg#icon-exercises-arrow"></use>`
Chang all paths.